pylib/Tools/Build/Shell.py

```diff
@@ -51,6 +51,7 @@ def __init__(self):
         self.options['allocate_cmd'] = (None, "Command to use for allocating nodes from the resource manager")
         self.options['deallocate_cmd'] = (None, "Command to use for deallocating nodes from the resource manager")
         self.options['asis_target'] = (None, "Specifies name of asis_target being built. This is used with \"ASIS\" keyword to determine whether to do anything.")
+        self.options['shell_mode'] = ("false", "Use shlex or shell-mode for parsing?")
```

Comment:
Why not default to the boolean False instead of "false"? That way parseOptions will correctly translate the value to a boolean for you.

Reply:
Hmm I didn't know about this parseOptions feature. I was using "lower()" to parse the options myself.

Are you saying that with boolean False, if I type "false" in the INI file it would still parse to be false as a boolean? I was thinking that it would give me a string "False" which would evaluate to True...

Reply:
Yep, when you call parseOptions it calls `__convert_value` which converts based on the type of the default value.
https://github.com/open-mpi/mtt/blob/2451cb18676b7e31ea19829173f09fb1f5fb53d7/pylib/System/TestDef.py#L81-L91